src/Search/FullTextSearch.php

```diff
@@ -143,4 +155,24 @@ public static function force_index_list()
             self::$all_indexes[$class] = $index;
         }
     }
+
+    /**
+     * Uses the disable_preindex_canview_check configuration to determine whether the given class should have the
+     * canView check applied.
+     *
+     * @param string $class The FQCN of the class to check
+     * @return bool
+     */
+    public static function isCanViewCheckDisabledForClass(string $class): bool
```

Comment:
function name reads kind of weird, `canViewCheckDisabledForClass()` might be better